style.css

```diff
@@ -0,0 +1,131 @@
+* {
+    box-sizing: border-box;
+
+  }
+  .Portfolio{
```

Comment:
.Portfolio should be renamed to .portfolio ( try to always match the naming convention used throughout the project ), e.g below you use .about and .experience classes ( starts with lowercase letter ), but .Portfolio class starts with uppercase.